src/helpers/alert.ts

```diff
@@ -8,7 +10,7 @@ const WrappedAlert = {
     buttons?: AlertButton[],
     options?: AlertOptions
   ) => {
-    if (android && !buttons && !options) {
+    if (android && !buttons && !options && IS_TESTING === 'false') {
```

Comment:
```suggestion
    if (android && !buttons && !options && !IS_TEST) {
```

Reply:
thanks! I will apply this immediately in the follow-up branch